great_expectations/data_context/store/html_site_store.py

```diff
@@ -204,6 +208,13 @@ def __init__(self, store_backend=None, runtime_environment=None):
         # can't necessarily set and list_keys like most other Stores.
         self.keys = set()
 
+        # Gather the call arguments of the present function (include the "module_name" and add the "class_name"), filter
+        # out the Falsy values, and set the instance "_config" variable equal to the resulting dictionary.
+        self._config = get_currently_executing_function_call_arguments(
+            include_module_name=True, **{"class_name": self.__class__.__name__,}
```

Comment:
This is a really elegant way to send in `kwargs` :) Thanks for helping me learn this